### DISCUSSION TITLE
💡【方法论】校招碎碎念 🆕

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#91] [Announcements]
https://imageslr.github.io/2021/autumn-recruit.html 

 前言

**Comment by zhangxu-ai** [comment]
反问问题总结的很好，可惜之前没看到o(￣▽￣)ｄ

**Comment by imageslr** [comment]
感谢反馈。请问你看到这篇文章的来源是哪里呢？ @zhangxu-ai

**Comment by zhangxu-ai** [comment]
> @imageslr
> 感谢反馈。请问你看到这篇文章的来源是哪里呢？ @zhangxu-ai

前两个月准备实习面试的时候google某个面试问题的时候看到的，之后一直跟着的文章进行学习准备的，非常感谢你的一系列文章，对我帮助很大🌹

**Comment by imageslr** [comment]
@zhangxu-ai 主要是我没怎么宣传过，所以想知道大家都是怎么发现的。Thanks！

**Comment by BlueStragglers** [comment]
感谢！东南大学计算机研一小白，今天搜索“后端面试会问计算机组成吗”，显示的第一篇博文就是您的。内容非常详尽，帮大忙了~

**Comment by imageslr** [comment]
> @BlueStragglers
> 感谢！东南大学计算机研一小白，今天搜索“后端面试会问计算机组成吗”，显示的第一篇博文就是您的。内容非常详尽，帮大忙了~

谢谢评论。我也搜了下，确实只有这个问题可以搜到hh，google 的权重设置真神奇

**Comment by wao3** [comment]
> @imageslr
> @zhangxu-ai 主要是我没怎么宣传过，所以想知道大家都是怎么发现的。Thanks！
> 
> 

google 搜索 select poll epoll，第一篇就是你的博客的文章！

**Comment by Shangguanguifen** [comment]
优秀优秀，文章也太详细了，博客内容很吸引人，看完感觉可以去面秋招了

**Comment by imageslr** [comment]
Ads Infra 2023 届后端实习开始内推啦，可转正，欢迎加入👏🏻 https://www.nowcoder.com/discuss/834943

**Comment by Tickdack** [comment]
神！真的是宝藏！可惜看到已经好迟了

**Comment by Johnny-Yh** [comment]
要是能早点看到这篇文章，字节实习三面应该不会挂了呜呜呜

**Comment by jingxuantju** [comment]
您好 想要一下这几个问题的模板 谢谢您
你如何选择不同 offer？/ 已经想好去哪个公司了吗？ / 有倾向吗？
你是不是投过别的公司？如果都要你怎么办？
你的长期职业规划是什么？1～3 年的规划是什么？
如果我录用你，你将怎样开展工作？
你认为其他同学如何评价你？/ 你觉得周围人眼中的你是什么样的？如果我问他们会得到一个什么样的评价？
麻烦您拉 再次感谢

**Comment by imageslr** [comment]
@jingxuantju 因为每个人情况不同，所以这里没有统一的回答模板，只能提供一个思路。

你如何选择不同 offer？/ 已经想好去哪个公司了吗？ / 有倾向吗？
> 如何选择不同 offer？上面有很多因素：城市、薪资、成长等。公司肯定说在争取当前的。


你是不是投过别的公司？如果都要你怎么办？
> 可以说其他公司**正在面试流程**，不要说已经拿到 offer。都要的话肯定来当前这个公司，但你应该给出几点具体的理由，直接说这个公司 HR 宣传的那些亮点就好。



你的长期职业规划是什么？1～3 年的规划是什么？
> 我自己的：短期融入团队；1~2 个月熟悉自己负责的项目，能承接需求；半年到一年深入了解自己负责的业务和架构，能够独立完成技术选型和开发；三年内，熟悉整个团队的业务和工作内容，能够完成大型项目的设计、分工、开发等；同时锻炼团队合作和项目管理能力。


如果我录用你，你将怎样开展工作？
> 同上，主要是围绕短期、1~2 个月、半年来陈述。如果你实习过，这个问题很好回答，想想你的 mentor 是怎么给你安排任务的即可。大部分都是 熟悉业务 → 熟悉环境 → 熟悉代码 → 熟悉架构 → 承接简单的需求 → 承接复杂的需求 → 总结，沉淀，复盘，改进... 的迭代流程。

你认为其他同学如何评价你？/ 你觉得周围人眼中的你是什么样的？如果我问他们会得到一个什么样的评价？
> 这个问题实际上是描述自己的优点和缺点。优点可以突出说一下 (比如文档能力、汇报能力、owner 意识等)，缺点不要说硬伤的 (可以说经验不足)。

**Comment by jingxuantju** [comment]
真的太感谢您啦 谢谢

**Comment by r6hk** [comment]
太干货了，谢谢博主分享
